platform/server/elasticsearch/Cluster.ts

```diff
@@ -13,7 +13,7 @@ export class Cluster {
     private readonly config: ElasticsearchConfig,
     logger: LoggerFactory
   ) {
-    this.log = logger.get('elasticsearch', 'client', config.clusterType);
+    this.log = logger.get(`elasticsearch::client::${config.clusterType}`);
```

Comment:
I've changed `LoggerFactory.get` signature in the latest commit, now it accepts just any number of string arguments as before, take a look. I still `join` parts inside the factory since it simplifies a lot of stuff, so going to keep it for now.